<?xml version="1.0"?>

<launch>

<node name="chess_manager" pkg="chess_moves" type="chess_manager.py" output ="screen">
</node>

</launch>